src/tables/Connectors.tsx

```diff
@@ -0,0 +1,117 @@
+    ];
+
+    return (
+        <Box
+            sx={{
+                mx: 2,
+            }}
+        >
+            {isFetchingConnectors ? (
+                <FormattedMessage id="common.loading" />
+            ) : null}
+
+            {fetchingConnectorsError ? { fetchingConnectorsError } : null}
+
+            {connectors.length > 0 ? (
+                <TableContainer component={Box}>
+                    <Typography>
+                        <FormattedMessage id="terms.connectors" />
+                    </Typography>
+                    <Table
+                        sx={{ minWidth: 350 }}
+                        aria-label={intl.formatMessage({
+                            id: 'connectors.title',
+                        })}
+                    >
+                        <TableHead>
+                            <TableRow
+                                sx={{
+                                    background: (theme) =>
+                                        theme.palette.background.default,
+                                }}
+                            >
+                                {columns.map((column, index) => {
+                                    return (
+                                        <TableCell
+                                            key={`${column.field}-${index}`}
+                                        >
+                                            <FormattedMessage
+                                                id={column.headerIntlKey}
+                                            />
+                                        </TableCell>
+                                    );
+                                })}
+                            </TableRow>
+                        </TableHead>
+                        <TableBody>
+                            {connectors.map((row, index) => (
+                                <TableRow
+                                    key={`Connector-${row.attributes.name}-${index}`}
+                                >
+                                    <TableCell>{row.attributes.name}</TableCell>
+                                    <TableCell>
+                                        {row.attributes.description}
+                                    </TableCell>
+                                    <TableCell>{row.attributes.type}</TableCell>
+                                    <TableCell>
+                                        {row.attributes.maintainer}
+                                    </TableCell>
+                                    <TableCell>
+                                        <FormattedDate
+                                            day="numeric"
+                                            month="long"
+                                            year="numeric"
+                                            value={row.attributes.updated_at}
+                                        />
+                                    </TableCell>
```

Comment:
The rows are still hardcoded and not using the column headers. This makes it much easier to have custom formatting for each piece of data... without just recreating the `datagrid` component from MUI.